Add Cosmos DB connector implementation (#1)
wso2-ballerina/module-cosmosdb: cosmosdb/tests/Ballerina.md

## Compatibility

| Ballerina Language Version  | Azure Cosmos DB API Version    |
| ----------------------------| -------------------------------|
|  0.990.0                    |   2017-02-22                   |

### Prerequisites

> 1. Create a database by visiting [Azure portal](https://portal.azure.com/)

Reviewer: ```suggestion
1. Create a database by visiting [Azure portal](https://portal.azure.com/).
```

Finish all sentences with a period.

2. Obtain the master key.

> Visit [here](https://docs.microsoft.com/en-us/rest/api/cosmos-db/access-control-on-cosmosdb-resources) for more information on obtaining master key.

Reviewer: ```suggestion
See the [topic on access control on Cosmos DB resources](https://docs.microsoft.com/en-us/rest/api/cosmos-db/access-control-on-cosmosdb-resources) for more information on obtaining a master key.
```

## Running Samples

> You can use the `tests.bal` file to test all the connector actions by following the below steps:

Reviewer: ```suggestion
You can use the `tests.bal` file to test all the connector actions by following the steps below:
```

> 1. Create ballerina.conf file in module-cosmosdb.

Reviewer: ```suggestion
1. Create a ballerina.conf file in module-cosmosdb directory.
```

> 2. Obtain the client Id, client secret, access token and refresh token as mentioned above and add those values in the ballerina.conf file.

Reviewer: ```suggestion
2. Obtain the client Id, client secret, access token, and refresh token as mentioned above and add those values in the ballerina.conf file.
```
